src/OpcApplicationConfiguration.cs

```diff
@@ -56,12 +57,31 @@ public static string Hostname
     /// Set the max string length the OPC stack supports.
     /// </summary>
     public static int OpcMaxStringLength { get; set; } = 4 * 1024 * 1024;
+
+    /// <summary>
+    /// Set when a flat directory certificate store should be used.
+    /// </summary>
+    public static bool EnableFlatDirectoryCertStore { get; set; } = false;
+
+    /// <summary>
+    /// the flat directory certificate store shall only be initialized once
+    /// </summary>
+    private static bool _flatDirectoryCertStoreInitialized = false;
 
     /// <summary>
     /// Configures all OPC stack settings.
     /// </summary>
     public async Task<ApplicationConfiguration> ConfigureAsync()
-    {
+    {
+        if (!_flatDirectoryCertStoreInitialized)
```

Comment:
isn't this ever called once?

Reply:
It should normally be only called once. In the unit tests it is called for every test though.

Reply:
btw: Would it be difficult to add unit tests for this? can be done in separate PR.